### PR TITLE
Restrict Wanderer fleets from entering Unfettered systems

### LINKED ISSUE #5329: Irrational Wanderer behaviour after the Unfettered invasion
**Describe the bug**
Wanderer ships fly to systems controlled by Unfettered after the Unfettered take some of Wanderer systems.

**Expected behavior**
Since Wanderers are peaceful, there is no reason for them to fly to Unfettered systems just to pointlessly fight (and die).

### PR DESCRIPTION
**Pull request by Amazinite**
**Content (Artwork / Missions / Jobs)**

## Summary
Gives the Wanderer government a travel restriction to avoid Unfettered systems. Wanderer defense fleet still sometimes spawn in Unfettered systems while they are fighting, but by the time the Arfecta appear, no Unfettered system has Wanderer defense fleet spawns anymore.

Resolves #5329.

## Save File
The Wanderers won't enter Unfettered systems from Ap'arak anymore. (This save still has the fleet spawns in Unfettered systems, though.)
[Test Pilot.txt](https://github.com/endless-sky/endless-sky/files/12797964/Test.Pilot.txt)

### CODE REVIEW

**Comment by Amazinite** [comment]
Opened as a draft because it doesn't look like it's actually working.

**Comment by Hurleveur** [comment]
What about the "wanderer drones" fleets?

**Comment by Amazinite** [comment]
What about them? Do they still spawn in the Unfettered systems?

**Comment by Hurleveur** [comment]
[Fighter Squadron.txt](https://github.com/endless-sky/endless-sky/files/12841583/Fighter.Squadron.txt)
my save says they do yeah, like `"Sich'ka'ara"` has extra drones? it was from the latest version but not including these changes ofc
mb they dont actually spawn tho bc they dont have a planet to spawn from? in that case we can clean it up later

**Comment by Amazinite** [comment]
I'll check. Those drones should stop spawning as soon as the Unfettered take the system.

**Comment by Hurleveur** [comment]
I just went to the UHai acquired systems and can confirm no drones spawn there, probably given to code preventing them from doing so

**Comment by Amazinite** [comment]
None of the systems have drone fleets defined after the Unfettered capture them.

File: data/wanderer/wanderers start.txt [hunk 1]
```diff
@@ -2926,3 +2930,13 @@ mission "Wanderers Ap'arak 3"
 		event "wanderers: spera anatrusk colony" 40
 		conversation
 			`A few minutes after you land on <planet>, a sudden uproar passes through the Wanderers in the spaceport. It's hard to be sure, but you think they sound excited, not frightened. Ships start taking off, more and more of them until the spaceport is almost deserted, and no one who is left seems at all interested in talking with you or explaining what is going on. You're probably going to have to return to your ship and see for yourself what all the fuss is about.`
+
+# Withdraw Wanderer defense fleets from Unfettered systems on pre-0.10.4 pilots.
+mission "Wanderers Retreat Patch"
+	landing
+	invisible
+	to offer
+		has "Wanderers Ap'arak 2: done"
```
Comment (Anarchist2) [inline data/wanderer/wanderers start.txt:2939]: Won't this mission also trigger on new save files as well?

File: data/wanderer/wanderers start.txt [hunk 2]
```diff
@@ -2926,3 +2930,13 @@ mission "Wanderers Ap'arak 3"
 		event "wanderers: spera anatrusk colony" 40
 		conversation
 			`A few minutes after you land on <planet>, a sudden uproar passes through the Wanderers in the spaceport. It's hard to be sure, but you think they sound excited, not frightened. Ships start taking off, more and more of them until the spaceport is almost deserted, and no one who is left seems at all interested in talking with you or explaining what is going on. You're probably going to have to return to your ship and see for yourself what all the fuss is about.`
+
+# Withdraw Wanderer defense fleets from Unfettered systems on pre-0.10.4 pilots.
```
Comment (Anarchist2) [inline data/wanderer/wanderers start.txt:2934]:
```suggestion
# Withdraw Wanderer defense fleets from Unfettered systems on pre-0.10.5 pilots.
```

File: data/wanderer/wanderers start.txt (hunk 1 above)
Comment (Amazinite) [reply data/wanderer/wanderers start.txt:2939]: Yes, but there's no harm in calling `remove` on a fleet twice. I also can't think of how we'd cause this to only offer on old save files.

**Comment by warp-core** [comment]
The feature it seeks to use is not currently working correctly. See #9904 for more details.

**Comment by ziproot** [comment]
Could this be accomplished by only restricting fleets jumping into systems that are more than one jump away from a wanderer system?

**Pull request closed without merging**